scripts/upgrades/view_proposal_status.sh

```diff
@@ -0,0 +1,8 @@
+STRIDED=scripts-local/upgrades/binaries/strided1
```

Comment:
what does this file do? It might be good to add a comment!

Reply:
its a lazy way to view the status of the proposal as you're waiting. I'm glad you called it out though, I'm just going to add this into submit_upgrades.sh since the voting period is so short